app/assets/javascripts/components/organization_display.jsx

```diff
@@ -28,12 +28,13 @@ var OrganizationsDisplay = React.createClass({
                 <option value="3"> 50 or more </option>
               </select>
               <br />
-              <div class="container-fluid text-center">
+              <br />
```

Comment:
Please use a `<div class="clearfix"></div>` instead. Also please replace all the `class` in jsx below to `className`